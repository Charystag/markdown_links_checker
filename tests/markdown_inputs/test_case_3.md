# Test Case 3: Valid URLs with Anchors

This test case includes valid URLs with anchors.

## Example 1: Links with Anchors to Sections

- [Markdown Guide Overview](https://www.markdownguide.org/#overview)
- [GitHub Docs Readme](https://docs.github.com/en#readme)

## Example 2: Links to Specific Sections

- [Python Docs FAQ](https://docs.python.org/3/faq/programming.html#how-do-i-use-strings)
- [Mozilla MDN Section](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Introduction#overview)

## Example 3: With parenthesis

- [Embedded parenthesis](https://en.wikipedia.org/wiki/Function_\(mathematics\))
