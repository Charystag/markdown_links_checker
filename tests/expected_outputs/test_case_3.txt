Checking file: `test_case_3.md'
[0;32mLine: `7'. url: `https://www.markdownguide.org/#overview' is valid. response_code: `200'[0m
[0;32mLine: `8'. url: `https://docs.github.com/en#readme' is valid. response_code: `200'[0m
[0;32mLine: `12'. url: `https://docs.python.org/3/faq/programming.html#how-do-i-use-strings' is valid. response_code: `200'[0m
[0;32mLine: `13'. url: `https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Introduction#overview' is valid. response_code: `200'[0m
[0;31mError on line : `17'. url: `https://en.wikipedia.org/wiki/Function_\(mathematics\'. response_code: `404'[0m
